### DISCUSSION TITLE
With Yahoo Finance Plus account, is there a way to extract more than 4 years of financial statement using yfinance ?

### DISCUSSION THREAD

**Discussion by TRokieG** [ranaroussi/yfinance#1655] [Development]
Hi dear yfinance community,

 I am new to yfinance, and I have tried to pull down company's financial statement data using yfinance. However, for my personal interest I want to load more than last 4 year's of balance sheet statement, so I bought a yfinance plus account, which on yahoo finance's website allows me to view more than last 4 year's company financial statement. 

My question is: is there a way we can **input** our user info into yfinance api, so it recognizes me as a yahoo finance plus account. As a result any fundamental data pulling will give me more than 4 years of data? 

I don't understand request or cookie very well, but my intuition tells me that since yfinance is built on as a web scrapper and for the `yf.Ticker()` function there is a parameter `session` in it, so there should be a way to pass in my authentication or cookie to make this work.  

Appreciate your attention in advance. Thanks!

**Comment by jab2727** [comment]
Hey there, did you ever find a solution to this? I too would like additional years of financials.
